content/en/docs/reference/command-line-tools-reference/feature-gates-removed/index.md

```diff
@@ -0,0 +1,539 @@
+| `SupportNodePidsLimit` | `true` | Beta | 1.15 | 1.19 |
+| `SupportNodePidsLimit` | `true` | GA | 1.20 | 1.23 |
+| `SupportPodPidsLimit` | `false` | Alpha | 1.10 | 1.13 |
+| `SupportPodPidsLimit` | `true` | Beta | 1.14 | 1.19 |
+| `SupportPodPidsLimit` | `true` | GA | 1.20 | 1.23 |
+| `SuspendJob` | `false` | Alpha | 1.21 | 1.21 |
+| `SuspendJob` | `true` | Beta | 1.22 | 1.23 |
+| `SuspendJob` | `true` | GA | 1.24 | 1.25 |
+| `Sysctls` | `true` | Beta | 1.11 | 1.20 |
+| `Sysctls` | `true` | GA | 1.21 | 1.22 |
+| `TTLAfterFinished` | `false` | Alpha | 1.12 | 1.20 |
+| `TTLAfterFinished` | `true` | Beta | 1.21 | 1.22 |
+| `TTLAfterFinished` | `true` | GA | 1.23 | 1.24 |
+| `TaintBasedEvictions` | `false` | Alpha | 1.6 | 1.12 |
+| `TaintBasedEvictions` | `true` | Beta | 1.13 | 1.17 |
+| `TaintBasedEvictions` | `true` | GA | 1.18 | 1.20 |
+| `TaintNodesByCondition` | `false` | Alpha | 1.8 | 1.11 |
+| `TaintNodesByCondition` | `true` | Beta | 1.12 | 1.16 |
+| `TaintNodesByCondition` | `true` | GA | 1.17 | 1.18 |
+| `TokenRequest` | `false` | Alpha | 1.10 | 1.11 |
+| `TokenRequest` | `true` | Beta | 1.12 | 1.19 |
+| `TokenRequest` | `true` | GA | 1.20 | 1.21 |
+| `TokenRequestProjection` | `false` | Alpha | 1.11 | 1.11 |
+| `TokenRequestProjection` | `true` | Beta | 1.12 | 1.19 |
+| `TokenRequestProjection` | `true` | GA | 1.20 | 1.21 |
+| `ValidateProxyRedirects` | `false` | Alpha | 1.12 | 1.13 |
+| `ValidateProxyRedirects` | `true` | Beta | 1.14 | 1.21 |
+| `ValidateProxyRedirects` | `true` | Deprecated | 1.22 | 1.24 |
+| `VolumePVCDataSource` | `false` | Alpha | 1.15 | 1.15 |
+| `VolumePVCDataSource` | `true` | Beta | 1.16 | 1.17 |
+| `VolumePVCDataSource` | `true` | GA | 1.18 | 1.21 |
+| `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
+| `VolumeScheduling` | `true` | Beta | 1.10 | 1.12 |
+| `VolumeScheduling` | `true` | GA | 1.13 | 1.16 |
+| `VolumeSnapshotDataSource` | `false` | Alpha | 1.12 | 1.16 |
+| `VolumeSnapshotDataSource` | `true` | Beta | 1.17 | 1.19 |
+| `VolumeSnapshotDataSource` | `true` | GA | 1.20 | 1.22 |
+| `VolumeSubpath` | `true` | GA | 1.10 | 1.24 |
+| `VolumeSubpathEnvExpansion` | `false` | Alpha | 1.14 | 1.14 |
+| `VolumeSubpathEnvExpansion` | `true` | Beta | 1.15 | 1.16 |
+| `VolumeSubpathEnvExpansion` | `true` | GA | 1.17 | 1.24 |
+| `WarningHeaders` | `true` | Beta | 1.19 | 1.21 |
+| `WarningHeaders` | `true` | GA | 1.22 | 1.24 |
+| `WindowsEndpointSliceProxying` | `false` | Alpha | 1.19 | 1.20 |
+| `WindowsEndpointSliceProxying` | `true` | Beta | 1.21 | 1.21 |
+| `WindowsEndpointSliceProxying` | `true` | GA | 1.22| 1.24 |
+| `WindowsGMSA` | `false` | Alpha | 1.14 | 1.15 |
+| `WindowsGMSA` | `true` | Beta | 1.16 | 1.17 |
+| `WindowsGMSA` | `true` | GA | 1.18 | 1.20 |
+| `WindowsRunAsUserName` | `false` | Alpha | 1.16 | 1.16 |
+| `WindowsRunAsUserName` | `true` | Beta | 1.17 | 1.17 |
+| `WindowsRunAsUserName` | `true` | GA | 1.18 | 1.20 |
+{{< /table >}}
+
+## Descriptions for removed feature gates
+
+
```

Comment:
nit: extra line

Reply:
👋 @sftim , I'm trying to describe how an author would create or edit feature-gate content.

These changes are only for feature gates that have reached the stage where the gate is removed?

What does an author do?
- The author edits the index page and updates the table on the page.
- The author edits/creates a feature-gate Markdown file and then includes the `shortcode` for the feature gate on the page (in what order). The feature-gate description is removed from the Alpha/Beta feature gate page to individual pages and applied through a `shortcode`.
- The author has to clean up another table on another page?

Reply:
> What does an author do (there are probably two work streams)?
> 
> * The author edits the index page and updates the Markdown table on the page.

Yes, that's right.
 
> * The author edits/creates a feature-gate description file and then includes or removes the `shortcode` for the feature gate on one of the feature-gate pages (in what order in the list of feature gates).

Creating a new description file is only needed the first time we add a feature gate (so not for promotion or removal). The order is always collated based on US English collation order.

---

> Does this new process help authors or help manage the content (reuse the descriptions)? Is a `shortcode` with individual Markdown files easier to manage (for authors) than data (JSON or YAML)?

- At this point, when you promote or deprecate or remove a feature gate the individual description file does not change. The index pages do change.
- If you're not sure about the sort order of the list of feature gate descriptions, you can now use your text editor to sort them.
- If you misspell a feature gate name _in the list of descriptions_ the site build picks it up.
- It's an enabling change. We can make **further changes**
  - for example, if we add a `removed: true` entry to the front matter of removed feature gates, we can change the feature description list to something like:
    ```
    {{< feature-gate-description-list removed="false" >}}
    ```
  - for example, if we add a some other data into the front matter of all feature gates, we can change the feature availability table to something like:
    ```
    {{< feature-gate-table selection="alpha,beta" >}}
    ```

>  Is a shortcode with individual Markdown files easier to manage (for authors) than data (JSON or YAML)?

IMO, putting the text in Markdown file will be easier than putting it in a JSON, YAML or TOML file - especially for localization teams. Localization teams already know how to localize Markdown.
We can even handle partial localization; imagine that a localization team has localized 90% of the gates but not the newest 10%. Feasibly we can render a page for that localization that mentions all the gates, 90% of the explanations are in the right language and 10% are in English with a little marker to call out that no localized explanation is available yet.